src/krux/pages/datum_tool.py

```diff
@@ -422,7 +422,7 @@ def view_qr(self):
             # when not sensitive, allow export to sd
             kvargs = {}
             if not self.sensitive:
-                kvargs = {"allow_export": True}
+                kvargs = {"allow_export": True, "transcript_tools": False}
```

Comment:
Since the condition here is about whether or not contents are sensitive, to disallow saving what might be a mnemonic or mnemonic entropy to sdcard, we might have a len(contents) check separately to decide whether or not transcript_tools are offered... for passphrases and encryption-keys that don't look "sensitive".